### ISSUE TITLE
updates (esp. of zarith)

### ISSUE THREAD

**Issue by hannesm** [ocaml-opam/opam-repository-mingw#19]
Dear Madam or Sir,

thank you so much for maintaining this repository - which I only discovered since I use a "setup-ocaml" GitHub action with a Windows runner.

I encountered an issue, though, since I need some functionality of zarith 1.13 (released July 2023), the Windows GitHub action fails to finish. This repository does only include zarith 1.12.

Now, I read through your "Updating" section in the README, but am still struggling what to do. Is it viable to PR just the single zarith 1.13 opam file / package, or is there something bigger needed (a merge of the current state or July 2023 state of opam-repository)?

Thanks for letting me know and again keep up the amazing work to get OCaml on the Windows Desktop! :)

Thanks for reading.

**Comment by hhugo** [comment]
If you configure the repo as advertised in the readme you should be able to install the latest version of zarith.

```
uses: ocaml/setup-ocaml@v2
with:
  opam-repositories: |
    opam-repository-mingw: https://github.com/ocaml-opam/opam-repository-mingw.git#sunset
    default: https://github.com/ocaml/opam-repository.git

```

**Comment by hannesm** [comment]
Thanks @hhugo, it looks like I did not read carefully (or back in the days when I worked on the GitHub windows action this was not needed). Now I'm curious why "setup-ocaml" doesn't do that for windows runners automatically?

EDIT: why this is not the default for setup-ocaml is also written in the README here "This is not the default in setup-ocaml at present because newer versions of packages may not contain required patches"

I confirm that modiyfing the yaml indeed solves my issue, and will close it.
